packages/fxa-settings/src/lib/auth-errors/en-US.ftl

```diff
@@ -12,8 +12,8 @@ auth-error-110 = Invalid token
 #                          as part of the string.
 #                           (for example: "in 15 minutes")
 auth-error-114 = You’ve tried too many times. Please try again { $retryAfter }.
-auth-error-138 = Unverified session
+auth-error-138-2 = Confirmed session
```

Comment:
```suggestion
auth-error-138-2 = Unconfirmed session
```
@millsoper - Should this be Unconfirmed to match the previous string Unverified?

Reply:
🤦 It definitely should -- thank you!